### PR TITLE
[LIBCLOUD-668] Add volumes_attached to node.extra

### PR DESCRIPTION
**Pull request by allardhoeve**
Please add volumes_attached extra info

### CODE REVIEW

**Comment by allardhoeve** [comment]
@kami, I think Travis still misbehaves...

**Comment by Kami** [comment]
Looks like a lint issue. Did you sync up this branch with latest trunk?

**Comment by allardhoeve** [comment]
Yes. Rebased all of them after the pep8 fix.

File: libcloud/compute/drivers/openstack.py [hunk 1]
```diff
@@ -2020,8 +2021,8 @@ def _to_node(self, api_node):
                 key_name=api_node.get('key_name', None),
                 disk_config=api_node.get('OS-DCF:diskConfig', None),
                 config_drive=config_drive,
-                availability_zone=api_node.get('OS-EXT-AZ:availability_zone',
-                                               None),
```
Comment (Kami) [inline libcloud/compute/drivers/openstack.py:2024]: Next time please this as it was - we prefer to go with "explicit over implicit" and always include default value when using dict.get.

**Comment by Kami** [comment]
Merged into trunk. Thanks.

**Pull request closed without merging**